helm_deploy/hmpps-interventions-service/templates/cronjob-data-extractor-analytics.yaml

```diff
@@ -13,7 +13,7 @@ spec:
           - name: data-extractor-analytics
             image: ministryofjustice/data-engineering-data-extractor:v1
             imagePullPolicy: Always
-            args: ["extract_psql_all_tables_to_jsonl.sh && transfer_local_to_s3.sh"]
+            args: ["extract_table_names.py && extract_pg_jsonl_snapshot.py && transfer_local_to_s3.sh"]
```

Comment:
Happy to approve, but could you make these changes in https://github.com/ministryofjustice/hmpps-helm-charts/tree/main/charts/generic-data-analytics-extractor as well, please, as we have #1385 open to replace this file, which would then lose these changes

Reply:
thank you! please let me know the new version and I can fold that into #1385

Reply:
Here is the PR to reflect the above: https://github.com/ministryofjustice/hmpps-helm-charts/pull/85

Reply:
new version is 0.1.4

Reply:
thanks! I will fold that into #1385 and close this PR